src/marqo/core/vespa_schema.py

```diff
@@ -16,6 +18,21 @@ class VespaSchema(ABC):
         '-': '_01',
     }
 
+    _DISTANCE_METRIC_MAP = {
+        DistanceMetric.Euclidean: 'euclidean',
+        DistanceMetric.Angular: 'angular',
+        DistanceMetric.DotProduct: 'dotproduct',
+        DistanceMetric.PrenormalizedAngular: 'prenormalized-angular',
+        DistanceMetric.Geodegrees: 'geodegrees',
+        DistanceMetric.Hamming: 'hamming'
+    }
+
+    def get_distance_metric(self, marqo_distance_metric: DistanceMetric) -> str:
```

Comment:
Should this function start with `_` for internal use?

Reply:
Fixed!